indico/web/templates/forms/selectize_widget.html

```diff
@@ -4,7 +4,7 @@
 {% block html %}
     <div class="i-form-field-fixed-width" data-tooltip-anchor>
         <input id="{{ field.id }}" class="typeahead" type="text" name="{{ field.name }}"
-               {%- if field.data is not none -%}data-data="{{ field._value() | tojson | forceescape}}"{%- endif -%}
+               {%- if field.data is not none -%}data-data="{{ field._value() or [] | tojson | forceescape}}"{%- endif -%}
```

Comment:
```suggestion
               {%- if field.data is not none -%}data-data="{{ (field._value() or []) | tojson | forceescape}}"{%- endif -%}
```

Might not even work as expected otherwise...

Reply:
Thanks for quick review on a Sunday afternoon.